modules/rosa-cluster-classic/README.md

```diff
@@ -1,7 +1,103 @@
-# rosa-cluster-classic
+# ROSA Classic Cluster sub-module
 
 ## Introduction
-TODO
+This Terraform sub-module is designed to facilitate the provisioning and management of ROSA Classic clusters within AWS infrastructure. Prior to using this sub-module, ensure that AWS IAM roles and policies are already established within the account, along with the necessary OIDC configuration and provider settings.
```

Comment:
@ciaranRoche just making sure that you're comfortable to call that cluster type "classic". 
Maybe we should change the name to something more specific?

Reply:
As I understand, we are going to have two modules types here:

- rosa-classic-cluster
- rosa-hcp-cluster

@ciaranRoche @gdbranco please confirm this is the right terminology

Reply:
That should be it AFAIK, we can check with @arendej if required

Reply:
That is my understanding, rosa-classic-cluster and rosa-hcp-cluster.